roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | grep "disabled"
+  register: no_selinux
+  when: (ansible_os_family == "RedHat")
+  ignore_errors: yes
+
 - name: permit openvswitch_t type in selinux
   shell: >
       semanage permissive -a openvswitch_t
+  when: (ansible_os_family == "RedHat") and not (no_selinux.stdout)
```

Comment:
If we make the above change, we can also change the check as `no_selinux.stdout|match( "SELinux status:.*enabled"`. This might be a bit more readable as well.